.github/workflows/publish.yaml

```
@@ -0,0 +1,23 @@
+name: Publish to PyPI
+on:
+  push:
+    tags:
+      - "*.*.*"
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
```

Reason not to use v2
```suggestion
        uses: actions/setup-python@v2
```

---

Thanks. I'll fix this.